server/lib/dbSeveralDocument.js

```diff
@@ -62,8 +71,8 @@ async function getCourseSemesterUsedRounds(courseCode, semester) {
     const finalObj = {
       usedRoundsThisSemester: [],
     }
-    await webBasedMemos.map(({ ladokRoundIds }) => finalObj.usedRoundsThisSemester.push(...ladokRoundIds))
-    await dbMigratedPdfs.map(({ koppsRoundId }) => finalObj.usedRoundsThisSemester.push(...koppsRoundId))
+    await webBasedMemos.map(({ applicationCodes }) => finalObj.usedRoundsThisSemester.push(...applicationCodes))
+    await dbMigratedPdfs.map(({ applicationCodes }) => finalObj.usedRoundsThisSemester.push(...applicationCodes))
```

Comment:
here it looks like it should destruct dbMigratedPdfs `applicationCode` (singular) if to compare with prev code